test/ThumbnailSpec.js

```diff
@@ -58,4 +65,25 @@ describe('<Thumbnail>', () => {
     );
     assert.ok(ReactDOM.findDOMNode(instance).lastChild.className.match(/\bcaption\b/));
   });
+
+  it('Should have an img with an onError callback', () => {
+    const onErrorSpy = sinon.spy();
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Thumbnail src="#" href="#" onLoad={() => {}} onError={onErrorSpy} />
+    );
+    let img = ReactTestUtils.findRenderedDOMComponentWithTag(instance, 'img');
+    ReactTestUtils.Simulate.error(img);
+    assert.ok(onErrorSpy.calledOnce);
```

Comment:
I think in general `expect` is nicer. It wouldn't be worth the effort to replace all `assert` instances, but it'd be good to use `expect` for new code.